x-pack/plugins/maps/public/lens/choropleth_chart/suggestions.ts

```diff
@@ -9,8 +9,8 @@ import { partition } from 'lodash';
 import { i18n } from '@kbn/i18n';
 import type { FileLayer } from '@elastic/ems-client';
 import type { SuggestionRequest, VisualizationSuggestion } from '@kbn/lens-plugin/public';
+import { RegionMap } from '@kbn/lens-icons';
```

Comment:
Other lens icon names are prefixed with `LensIcon`. Mind changing name to `LensIconRegionMap`?

Reply:
Good idea!